desci-server/src/routes/v1/doi.ts

```diff
@@ -1,11 +1,21 @@
 import { Router } from 'express';
 
-import { asyncHander, checkMintability, ensureNodeAccess, ensureUser, getDoi, mintDoi } from '../../internal.js';
-
+import {
+  asyncHander,
+  attachDoi,
+  attachDoiSchema,
+  checkMintability,
+  ensureNodeAccess,
+  ensureUser,
+  getDoi,
+  mintDoi,
+  validate,
+} from '../../internal.js';
 const router = Router();
 
 router.post('/check/:uuid', [ensureUser, ensureNodeAccess], asyncHander(checkMintability));
 router.post('/mint/:uuid', [ensureUser, ensureNodeAccess], asyncHander(mintDoi));
+router.post('/attach', [ensureUser, ensureNodeAccess, validate(attachDoiSchema)], asyncHander(attachDoi));
```

Comment:
Not immediately clear what “attach” means in this case :thinking:

Reply:
updates the manuscript in the manifest to add doi to the payload